src/K1DModel/tendency.jl

```diff
@@ -55,8 +56,23 @@ end
         ),
     )
 
-    @. S_Nl = ifelse(S_liq < 0 || isnan(N_act), FT(0), max(FT(0), N_act - (N_aer_0 - Y.N_aer)) / dt)
+    @. S_Nl = ifelse(
+        S_liq < 0 || isnan(N_act),
+        FT(0),
+        max(FT(0), N_act - N_liq) / dt
```

Comment:
`N_aer_0 - N_aer` differs from `N_liq`. The first is the number of the activated particles, while `N_liq` is the number of liquid droplets that might be reducing due to microphysical processes.

Reply:
Right. But I think using `N_liq` is the correct one? - If the number of cloud droplets is reduced because of sedimentation or collisions, then we have the potential to activate more particles. Similarly, if the number of cloud droplets has increased, then I should not be activating new particles on top of that 

I think we should have one variable for background (aka unactivated, aka interstitial) aerosol `N_aer` and one variable for cloud droplet number concentration `N_liq`. Maybe there is no need for `N_aer_0`